downloader/download_manager.py

```diff
@@ -0,0 +1,9 @@
+from downloader.history_downloader.history_manager import HistoryDownloader
+from downloader.history_saver.save_manager import HistorySaver
+
+bitcon_symbol = 'BTC/USDT'
+exchange_name = 'binance'
+bitcon_history = HistoryDownloader(exchange_name=exchange_name, crypto_symbol=bitcon_symbol).get_history()

```

Comment:
It is much clearer to initialize an object separetly from calling its method ;)

Reply:
Fixed 👍